far_mar.rb

```diff
@@ -0,0 +1,11 @@
+require 'csv'
+
+module FarMar
+
+end
+
+#might not need these, below...
+# require 'lib/farmar_market'
```

Comment:
It's be good to require the data classes here. That way, you can require just this one file and get access to all the modules you'll need.  See my note about requiring this file in the spec_helper as an example.